webpack/webpack.config.prod.js

```diff
@@ -58,7 +58,7 @@ export default {
     // new webpack.optimize.AggressiveMergingPlugin(),
     // new webpack.optimize.MinChunkSizePlugin({minChunkSize: 50000}),
     // new webpack.optimize.UglifyJsPlugin({compressor: {warnings: false}, comments: /(?:)/}),
-    new AssetsPlugin({path: path.join(root, 'build'), filename: 'assets.json'}),
+    new AssetsPlugin({path: path.join(root, 'build'), filename: 'assets.json', includeManifest: true}),
```

Comment:
that `includeManifest` is my baby right there. It allows us to include the webpack boostrapper in the HTML skeleton without perform a read during runtime. That mean we don't have an async middleware handler, which means our server throws errors again! yay!